cmd/sops/subcommand/exec/exec.go

```diff
@@ -0,0 +1,100 @@
+package exec
+
+import (
+	"log"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+type ExecOpts struct {
+	Command string
+	Plaintext []byte
+	Background bool
+	Fifo bool
+	User string
+}
+
+func GetFile(dir string) *os.File {
+	handle, err := ioutil.TempFile(dir, "tmp-file")
+	if err != nil {
+		log.Fatal(err)
+	}
+	return handle
+}
+
+func ExecWithFile(opts ExecOpts) {
+	if opts.User != "" {
+		SwitchUser(opts.User)
+	}
+
+	if runtime.GOOS == "windows" && opts.Fifo {
+		log.Print("no fifos on windows, use --no-fifo next time")
+		opts.Fifo = false
+	}
+
+	dir, err := ioutil.TempDir("/tmp/", ".sops")
```

Comment:
using `/tmp/` here will break Windows users. Can you use the `""` empty string to allow os dependant temp location ?

Reply:
good catch, will do.